openlibrary/plugins/upstream/mybooks.py

```diff
@@ -287,13 +287,13 @@ def get_shelf(name, page=1):
             # Ideally, do all 3 lookups in one add_availability call
             want_to_read.docs = add_availability(
                 [d for d in want_to_read.docs if d.get('title')]
-            )[:4]
+            )[:5]
```

Comment:
the value [5] may conflict with what's needed on desktop -- we may need a special solution on the frontend for desktop to hide the 5th item so there isn't a scrollbar for each carousel.